common/src/main/java/org/apache/nemo/common/coder/BytesDecoderFactory.java

```diff
@@ -84,7 +88,7 @@ private BytesDecoder(final InputStream inputStream) {
           returnedArray = true;
           return new byte[0];
         } else {
-          throw new IOException("EoF (empty partition)!"); // TODO #120: use EOF exception instead of IOException.
+          throw new EOFException("EoF (empty partition)!"); // TODO #120: use EOF exception instead of IOException.
```

Comment:
Remove TODO?

Reply:
I'm not sure it resolves the issue. @seojangho Do you think it resolves issue 120?